varnish-http-purge.php

```diff
@@ -35,7 +35,7 @@ class VarnishPurger {
 	 * Version Number
 	 * @var string
 	 */
-	public static $version = '4.8.1';
+	public static $version = '4.9';
```

Comment:
Version mismatch with header
```suggestion
	public static $version = '5.0';
```